build-tools/src/test/groovy/org/elasticsearch/gradle/plugin/scanner/test_classes/ExtensibleClass.java

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
We really do not want java classes in src/(main test)/groovy as joint compilation is ridicolous slow. You can either just rename those classes to groovy or can you move them into testFixtures/java? Ultimately they are test fixtures aren't they?